### ISSUE TITLE
A Segmentation fault  occurred while connecting to milvus

### ISSUE THREAD

**Issue by Yassin-fan** [milvus-io/milvus#5377]
**A Segmentation fault  occurred while connecting to milvus **

**Describe the bug**
I use ubuntu18 to run milvus_cpu with docker and the log shows that Milvus server started successfully. 
Then I use example.py to have a try.( Python 3.6.9 and pymilvus-1.1.0 )
And it always shows that Segmentation fault.
I have tried a lot and have no idea why.
By the way, I tried on ubuntu16 to run milvus_cpu and milvus_gpu and get the same fault .

**Steps/Code to reproduce behavior**
Use ubuntu18 to run milvus_cpu with docker 
The log shows that Milvus server started successfully. 
Use example.py to have a try.( Python 3.6.9 and pymilvus-1.1.0 )
Always shows that Segmentation fault.

By the way, I tried on ubuntu16 to run milvus_cpu and milvus_gpu and get the same fault .

**Expected behavior**
Hope to run connect with the server.

**Method of installation**
- [√ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Ubuntu18
- Python 3.6.9 
- pymilvus-1.1.0

- Milvus version (master or released version)
1.1.0

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`
I did not change it while using milvus_cpu and change gpu part while trying milvus_gpu

```yaml
paste-file-content-here
```

**Screenshots**
![image](https://user-images.githubusercontent.com/34295915/119356067-95f8cb00-bcd8-11eb-8d74-ad1db6df918d.png)

**Additional context**

**Comment by Yassin-fan** [comment]
> I had the same problem
> ![image](https://user-images.githubusercontent.com/58025953/119360501-e1a97580-bcd4-11eb-8e59-fd9adbb0d85f.png)

Actually I have to give it up and use faiss, for I cannot find how to solve this problem.

**Comment by yhmo** [comment]
Link to https://github.com/milvus-io/pymilvus/issues/517
This issue can be fixed by install higher grpc lib version:
```
grpcio==1.22.0
grpcio-tools==1.22.0
```

**Comment by Yassin-fan** [comment]
> Link to [milvus-io/pymilvus#517](https://github.com/milvus-io/pymilvus/issues/517)
> This issue can be fixed by install higher grpc lib version:
> 
> ```
> grpcio==1.22.0
> grpcio-tools==1.22.0
> ```

Thank you so much, I have connected with milvus server!
By the way, I suppose that the doc of milvus should add this tip in.

**Comment by yhmo** [comment]
An issue has been created for adding this into milvus documentation:
https://github.com/milvus-io/milvus/issues/5391

**Comment by yhmo** [comment]
The milvus project use a grpc project diverse from the official grpc master branch, in Dec 2019:
https://github.com/milvus-io/grpc-milvus

The custom project removed some third-parties and fix one issue(I am not sure). It could not match with too old/new proto version I guess.

**Comment by XuanYang-cn** [comment]
I think this's can be fixed in pymilvus.

Altering pymilvus's requirements lib version in `setup.py` can prevent that, thanks @yhmo  for investigating.

**Comment by yhmo** [comment]
> I think this's can be fixed in pymilvus.
> 
> Altering pymilvus's requirements lib version in `setup.py` can prevent that, thanks @yhmo for investigating.

That would be great.

**Comment by yhmo** [comment]
Solved by pymilvus 1.1.1
